MBBSEmu/HostProcess/MbbsHost.cs

```diff
@@ -217,6 +217,10 @@ private void WorkerThread()
                         {
                             session.Status = 1;
                             session.InputBuffer.SetLength(0);
+                            
+                            //Redisplay Main Menu prompt after global
```

Comment:
We should split this display out from `MainMenuDisplay()` into a new method with a new state such as `MainMenuInputDisplay()` that displays this string. This way we can set the user state here to display it and we're not duplicating the string.

Additionally it'd let us perhaps add the ability for someone to implement a custom version using a specified ANSI in a future enhancement.

Reply:
Great idea... this is done.